src/scripts/helpers.js

```diff
@@ -0,0 +1,66 @@
+export function rotateField(tabel, size) {
+  const rotatedField = [];
+
+  for (let c = 0; c < size; c++) {
+    const row = [tabel[0][c], tabel[1][c], tabel[2][c], tabel[3][c]];
+    rotatedField.push(row);
+  }
+
+  return rotatedField;
+};
+
+export function checkPossibleMoves(field, size) {
+  let possibleMoves = [];
+
+  for (let r = 0; r < size; r++) {
+    for (let c = 0; c < size; c++) {
```

Comment:
better to use full word for variables - `row`, `column`
it will increase readibility